### DISCUSSION TITLE
how to check matcher in rule order

### DISCUSSION THREAD

**Discussion by passwa11** [projectdiscovery/nuclei#5123] [Q&A] [answered]
body: bbbb ccccc aaaa 

contains_all(body, 'aaaa', 'bbbb', 'cccc')

false matchers now will search all strings ?

**Comment by GeorginaReeder** [comment, marked as answer]
Hey there @passwa11 , thanks for your question :)

Just to check while we investigate this - what are you looking to achieve exactly? We have the `contains_any` function, is that what you want?

Let us know!
